src/goto-instrument/dump_c.cpp

```diff
@@ -1236,18 +1232,15 @@ void dump_ct::cleanup_expr(exprt &expr)
 
     PRECONDITION(old_components.size()==old_ops.size());
     exprt::operandst::iterator o_it=old_ops.begin();
-    for(struct_union_typet::componentst::const_iterator
-        it=old_components.begin();
-        it!=old_components.end();
-        ++it)
+    for(const auto &c : old_components)
```

Comment:
Suggest `c` -> `old_comp` to more clearly distinguish it from the also-in-scope `comp`